lib.php

```diff
@@ -335,7 +335,15 @@ function local_annoto_extend_settings_navigation(settings_navigation $settingsna
 
     // Check and create LTI external tool.
     require_once($CFG->dirroot . '/mod/lti/locallib.php');
-    $lti = lti_get_tool_by_url_match($settings->toolurl);
+    // Will return all available tools by domain from toolurl.
+    $possibletools = lti_get_tools_by_url($settings->toolurl, LTI_TOOL_STATE_CONFIGURED, null);
```

Comment:
does this work for LTI 1.3 without changing ?
<img width="753" alt="image" src="https://github.com/Annoto/moodle-local_annoto/assets/6747147/5d796c9c-48ee-4468-92ec-bc2ed162a7e5">

Reply:
Updated to use domain instead